prow/manifests/overlays/metal3/config.yaml

```diff
@@ -1143,6 +1217,21 @@ presubmits:
     - name: generate
       branches:
         - main
+      skip_if_only_changed: '(((^|/)OWNERS)|(\.md))$'
```

Comment:
cannot we filter all .md files (even README )

Reply:
Can we take this as it is for now? I will handle this and `imagePullPolicy:IfNotPresent` in a separate PR

Reply:
NP